packages/app/navigation/root-stack-navigator.tsx

```diff
@@ -45,14 +46,17 @@ export function RootStackNavigator() {
         component={BottomTabNavigator}
         options={{ headerShown: false }}
       />
-
-      {/* Screens accessible in most of the navigators */}
-      <Stack.Group screenOptions={screenOptions({ safeAreaTop, isDark })}>
+      {/* Screens without default header */}
```

Comment:
if some screens need to without default header, add to this places.
alos next I need to change some screen to this places, I have tasks on Linear.